modules/web/js/ballerina/components/editable-text.jsx

```diff
@@ -30,7 +30,7 @@ class EditableText extends React.Component {
     }
 
     renderTextBox() {
-        const {x, y, width, height = 25, onChange, onBlur, onKeyDown, children,
+        const {x, y, width, height = 25, onChange, onBlur, onKeyDown, children="",
```

Comment:
Single qoutes seems to be added by now in our lint fix process. Children here should be a string. I'll add proptype validations as part of the remaining lint fixes.